<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackerWindow</class>
 <widget class="QMainWindow" name="TrackerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
background-color:qlineargradient(spread:pad, x1:0.025, y1:0.385909, x2:0.99, y2:0.374364, stop:0 rgba(255, 18, 3, 255), stop:1 rgba(76, 16, 255, 255));
border-radius: 10px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>981</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QListWidget" name="Tracker_List">
        <property name="font">
         <font>
          <family>Segoe UI Historic</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget{
background-color: rgb(203, 203, 203)
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="Link_Input">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>10</y>
       <width>981</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgb(203, 203, 203);
border-radius: 10px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding-top:5px;
text-align: center;
border-radius:15px
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
     </property>
     <property name="text">
      <string>Add product</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding-top:5px;
text-align: center;
border-radius:15px
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
     </property>
     <property name="text">
      <string>Save preset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoadButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding-top:5px;
text-align: center;
border-radius:15px
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
     </property>
     <property name="text">
      <string>Load preset</string>
     </property>
    </widget>
    <widget class="QLabel" name="Status_Label">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>50</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color:rgb(255, 170, 255)rgb(85, 0, 127)
}</string>
     </property>
     <property name="text">
      <string>Preset loaded!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>540</y>
       <width>981</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="Notif_List">
        <property name="styleSheet">
         <string notr="true">QListWidget{
background-color: rgb(203, 203, 203)
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="DeleteButton">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding-top:5px;
text-align: center;
border-radius:15px
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
     </property>
     <property name="text">
      <string>Delete preset</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
